### PR TITLE
WT-10883 Add to s_all to check for new lines in test/format and .sh files 

### PR DESCRIPTION
**Pull request by amiyuki7**
`dist/s_whitespace` was not checking failure config files and all sh scripts for whitespace formatting.

### CODE REVIEW

**Review by etienneptl — APPROVED** [review]
Quick win, love it, thanks!

File: dist/s_whitespace [hunk 1]
```diff
@@ -24,6 +24,8 @@ find bench dist examples ext src test \
     -name '*.dox' -o \
     -name '*.in' -o \
     -name 's_*' -o \
+    -name '*.sh' -o \
```
Comment (etienneptl) [inline dist/s_whitespace:27]: One np, can you have this list alphabetized, please?

**Review by ruby-oujo — APPROVED** [review]
Nice work!  LGTM 🙂

File: dist/s_whitespace (hunk 1 above)
Comment (amiyuki7) [reply dist/s_whitespace:26]: ah, sorry, I forgot I left this open. Done.

Comment (etienneptl) [reply dist/s_whitespace:26]: Thanks!

**Comment by etienneptl** [comment]
evergreen merge

**Pull request merged**